src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/NavigationDrawer.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Threading;
+using Xamarin.Forms;
+using ElmSharp;
+using ElmSharp.Wearable;
+using EWidget = ElmSharp.Widget;
+using ELayout = ElmSharp.Layout;
+using EColor = ElmSharp.Color;
+
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public class NavigationDrawer : ELayout
+    {
+        readonly int MinimumDrawerHeight = 50;
+
+        Box _gestureBox;
+        Box _outterBox;
+        Box _contentBox;
+        EvasObject _label;
+        GestureLayer _gestureLayer;
+        GestureLayer _dragGestureLayer;
+        EvasObject _parent;
+        IAnimatable _animatable;
+        bool _isOpen;
+
+        CancellationTokenSource _fadeInCancelTokenSource = null;
+
+        public bool IsOpen
+        {
+            get
+            {
+                return _isOpen;
+            }
+            set
+            {
+                if(_isOpen != value)
+                {
+                    if(value)
+                    {
+                        Open();
+                    }
+                    else
+                    {
+                        Close();
+                    }
+                }
+            }
+        }
+
+        public event EventHandler Toggled;
+
+        public NavigationDrawer(EvasObject parent, IAnimatable animatable) : base(parent)
```

Comment:
How about implement `IAnimatable` on this class?
``` c#
    public class NavigationDrawer : ELayout, IAnimatable 
```

Reply:
Fixed.